t/lib/t/MusicBrainz/Server/Controller/Artist/EditRelationships.pm

```diff
@@ -35,6 +35,7 @@ test 'Test adding a relationship' => sub {
         $mech->post('/artist/745c079d-374e-4436-9448-da92dedef3ce/edit', {
             'edit-artist.name' => 'Test Artist',
             'edit-artist.sort_name' => 'Artist, Test',
+            'edit-artist.comment' => 'Yet Another Test Artist',
```

Comment:
Hmm, this doesn't seem ideal.  I think it makes sense to ignore the field if it's omitted rather than assume it's being set to an empty string.  (I guess this probably won't cause issues on the website, but might for scripts that are submitting things manually.)

https://metacpan.org/dist/HTML-FormHandler/view/lib/HTML/FormHandler/Manual/Validation.pod says that the form's (and I assume the field's) `validate` method will always be called.  Instead of setting `validate_when_empty`, which seems to force the value to `''`, maybe we can implement a `validate` method instead (which you can set the value from, so we could check if it's set before running `trim_comment` on it).